src/services/routes/org.routes.ts

```diff
@@ -22,7 +36,6 @@ interface OrgUrls {
 const makeUrls = (name: string): OrgUrls => ({
   self: `/orgs/${name}`,
 });
-
```

Comment:
Nitpick: we probably should keep this empty line for consistency.

ESLint allows both approaches because `makeUrls` is a variable, and variables are allowed to precede a function in the even the function uses that variable. However, in this particular case, `makeUrls` shouldn't be a variable, but a literal function... but that's a separate conversation as that's outside the scope of this PR.

Reply:
change to function